index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="main-div">
+      <!--HEADER SECTION-->
+      <header class="header-section">
+        <div class="toolbar">
+            <a href="#">My Logo</a>
+            <nav class="nav-item">
+              <img src="images/toggle.png" alt="Toggle">
+            </nav>
+          </div>
+        </header>
+        
+    <!--HEADLINE SECTION-->
+    <section class="headline-section">
+        <h1 class="heading">I am Daisy <br/>Glad to see you!</h1>
+        <p class="profile">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="info">LET'S CONNECT</p>
+        <div class="media-icon">
+            <ul>
+                <li class="icon">
+                    <a href="#">
+                      <img src="images/twitter.png" alt="Twitter">
+                    </a>
+                </li>
+                <li class="icon">
+                    <a href="#">
+                      <img src="images/Linkedin.png" alt="LinkedIn">
+                    </a>
+                </li>
+                <li class="icon">
+                    <a href="#">
+                      <img src="images/media.png" alt="media">
+                    </a>
+                </li>
+                <li class="icon">
+                    <a href="#">
+                        <img src="images/github.png" alt="Github"></a>
+                </li>
+                <li class="icon">
+                    <a href="#">
+                      <img src="images/fingers.png" alt="Finger">
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+
+          <!--WORK section-->
+          <section class="work">
```

Comment:
-[ ] Here work section background color, border, the margin should match the original website.
- your website
![edit3](https://user-images.githubusercontent.com/26767383/149564713-7f7aca64-3e06-4553-a661-d153e54f4005.PNG)

- Original website  
![o3](https://user-images.githubusercontent.com/26767383/149564676-fd53d3b6-e888-432a-b1ae-9a89fa8f9202.PNG)